data/json/monsters/zed_ferrous.json

```
@@ -0,0 +1,278 @@
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_shell",
+    "type": "MONSTER",
+    "name": { "str": "rust shell zombie" },
+    "description": "This mangled body seems to develop a rusty spiked shell in its back.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "83500 g",
+    "hp": 200,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 0,
+    "vision_night": 3,
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "attack_effs": [ { "id": "tetanus", "chance": 10 } ],
+    "death_drops": "default_zombie_death_drops",
+    "death_function": [ "NORMAL" ],
+    "burn_into": "mon_zombie_scorched",
+    "upgrades": { "half_life": 15, "into": "mon_zombie_plated" },
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_siren",
+    "type": "MONSTER",
+    "name": { "str": "siren zombie" },
+    "description": "Some cruel will has formed this zombie into a living speaker system. It has no arms and it's head is covered in rust hiding most features, but it's lips have distended into a large cone that constantly emits a loud siren wail.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "83500 g",
+    "hp": 100,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "brown",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 0,
+    "vision_night": 3,
+    "harvest": "zombie",
+    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "GRAB", 7 ], [ "scratch", 20 ] ],
+    "death_drops": "default_zombie_death_drops",
+    "death_function": [ "NORMAL" ],
+    "burn_into": "mon_zombie_scorched",
+    "upgrades": { "half_life": 15, "into": "mon_zombie_plated" },
+    "flags": [
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "LOUDMOVES",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_plated",
+    "type": "MONSTER",
+    "name": { "str": "rust plated zombie" },
+    "description": "A slow moving statue of a zombie covered with a scaly layer of rust.  Each step clangs with the sound of metal.",
```

```suggestion
    "description": "Ruptures in this zombie's skin expose fragments of metal arranged like armor underneath.  Black sludge can be seen in places where the rust has receded.",
```
Going with this I'd increase the armor values on this one to 15ish. If we say there's some blob magic going on here to get purer metals from metal oxides stuck in zombies it would open some more interesting opportunities for evolution paths.